app/controllers/users_controller.rb

```diff
@@ -17,6 +17,13 @@ def create
 
     if @user.save
 
+      begin
+        Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') unless Rails.env.test?
```

Comment:
okay, what is the proper way to use it in the test environment? Something like this?

```
Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') 
if params[:stripeToken]
```